Gems/Terrain/Code/Source/TerrainRenderer/TerrainMeshManager.h

```diff
@@ -218,6 +219,7 @@ namespace Terrain
             AZ::RHI::DrawListTag m_drawListTag;
             AZ::RHI::Ptr<AZ::RHI::ShaderResourceGroupLayout> m_drawSrgLayout;
             AZ::RPI::ShaderVariant m_shaderVariant;
+            AZ::Name m_materialPipelineName;
```

Comment:
Where does this get set?

Reply:
m_cachedDrawData.push_back({ shader, shaderOptions, pipelineState, drawListTag, drawSrgLayout, variant, materialPipelineName });